OpenQA/Isotovideo/Utils.pm

```diff
@@ -128,7 +128,7 @@ configuration variables.
 sub handle_generated_assets ($command_handler, $clean_shutdown) {
     my $return_code = 0;
     # mark hard disks for upload if test finished
-    return unless $bmwqemu::vars{BACKEND} =~ m/^(qemu|generalhw)$/;
+    return undef unless $bmwqemu::backend->{backend}->can('do_extract_assets');
```

Comment:
In case the method is not implemented, this will return silently.
Before it would die.
Maybe at least a warning or info should be written in the log?

Reply:
Well, it would never die in practice because the backends live in the same codebase. Also, a log message here would be visible in every single test run unless this feature is implemented - do we want to implement it in all backends? Otherwise I'd consider this harmful.

Reply:
Ok, I should have noted that the warning should only appear if `PUBLISH_HDD_*` is set.
Not sure how we handle other cases where special vars trigger certain code.

Reply:
Ack. I'd consider that a new feature, though? And rather do it in another branch with a test to check the warning is logged.